### PR TITLE
[Datahub] [Redesign] Thumbail preview

### PR DESCRIPTION
**Pull request by cmoinier**
### Description

This PR introduces changes for the redesign of the thumbnail image. It includes the following changes : 

- [x] Move the contact & lightbox section up in the about section
- [x] Increase thumbnail size
- [x] e2e tests

### Architectural changes

Added `basiclightbox` library.

### Screenshots

image

### Quality Assurance Checklist

- [x] Commit history is devoid of any _merge commits_ and readable to facilitate reviews
- [x] If **new logic** ⚙️ is introduced: unit tests were added
- [x] If **new user stories** 🤏 are introduced: E2E tests were added
- [x] The [documentation website](docs) 📚 has received the love it deserves

---

**This work is sponsored by [GeoCat](https://www.geocat.admin.ch/fr/home/evenements.html)**.

### CODE REVIEW

File: apps/datahub/src/app/record/record-metadata/record-metadata.component.html [hunk 1]
```diff
@@ -23,16 +23,7 @@
...
-      <div>
-        <div *ngIf="metadataQualityDisplay">
-          <p class="text text-gray-700 text-xs mb-3 uppercase" translate>
-            record.metadata.quality
-          </p>
-          <gn-ui-metadata-quality
-            [metadata]="facade.metadata$ | async"
-            [metadataQualityDisplay]="metadataQualityDisplay"
-          ></gn-ui-metadata-quality>
-        </div>
```
Comment (jahow) [inline apps/datahub/src/app/record/record-metadata/record-metadata.component.html:35]: hmm, was there a reason to remove that? This indicator should stay somewhere on the page, although I agree that now, above the thumbnail is probably not the best.

Also our E2E tests are lacking because we're not testing that this indicator indeed show up :grimacing:

File: libs/ui/elements/src/lib/image-overlay-preview/image-overlay-preview.component.html [hunk 2]
```diff
@@ -1,29 +1,25 @@
-<gn-ui-content-ghost
+<div
+  *ngIf="imageUrl"
```
Comment (jahow) [inline libs/ui/elements/src/lib/image-overlay-preview/image-overlay-preview.component.html:2]: Looks like the "content-ghost" for the thumbnail is gone, could you please bring it back and see if your changes still work? thanks :)

File: libs/ui/elements/src/lib/metadata-contact/metadata-contact.component.html [hunk 3]
```diff
@@ -19,6 +19,7 @@
           class="text-primary font-title text-21 mr-2 cursor-pointer hover:underline"
           (click)="onOrganizationClick()"
           data-cy="organization-name"
+          *ngIf="shownOrganization?.name"
```
Comment (jahow) [inline libs/ui/elements/src/lib/metadata-contact/metadata-contact.component.html:22]: was that change intentional? I'm not sure effect it has but it looks not intended :wink:

**Review by jahow — COMMENTED** [review]
Thanks for the changes, I think some unintentional changes slipped through, could you try and see if you can narrow down what was really intended? thanks!

File: libs/ui/elements/src/lib/image-overlay-preview/image-overlay-preview.component.html (hunk 2 above)
Comment (cmoinier) [reply libs/ui/elements/src/lib/image-overlay-preview/image-overlay-preview.component.html:2]: I removed that because it was creating a weird display of the thumbnail, but if it's important I can look into having it back.

File: libs/ui/elements/src/lib/metadata-contact/metadata-contact.component.html (hunk 3 above)
Comment (cmoinier) [reply libs/ui/elements/src/lib/metadata-contact/metadata-contact.component.html:22]: It was intentional! I had a case where the organization had no name (or an error on it perhaps), so I added it as a fallback. I know it's not included in the PR, but figured it was shorter a fix than creating a ticket for it...

File: libs/ui/elements/src/lib/image-overlay-preview/image-overlay-preview.component.html (hunk 2 above)
Comment (jahow) [reply libs/ui/elements/src/lib/image-overlay-preview/image-overlay-preview.component.html:2]: hmm, maybe the content ghost doesn't work well on a colored background? what do you mean by "weird display"? The content ghost is important to avoid layout shifts when the data loading is finished

File: apps/datahub/src/app/record/record-metadata/record-metadata.component.html (hunk 1 above)
Comment (jahow) [reply apps/datahub/src/app/record/record-metadata/record-metadata.component.html:35]: Yes ok, I see what you mean. These kind of "hidden" components are tricky. This component is important for platforms which use it, but enabling it is not necessarily straightforward so we tend to forget them. See 

@lucilecollignon do you have a suggestion on what do to with this component? Currently it looks like this:
image

File: libs/ui/elements/src/lib/metadata-contact/metadata-contact.component.html (hunk 3 above)
Comment (jahow) [reply libs/ui/elements/src/lib/metadata-contact/metadata-contact.component.html:22]: ok, then I would suggest trying to identify what would cause an organization to have no name. This can have other implications so I think it'd be best if you could go ahead and create a bug ticket? thanks!

File: apps/datahub/src/app/record/record-metadata/record-metadata.component.html (hunk 1 above)
Comment (cmoinier) [reply apps/datahub/src/app/record/record-metadata/record-metadata.component.html:35]: Interesting, I never saw this component before :slightly_frowning_face: 

Waiting for Lucile's input, but I feel like it should go under the lightbox and contact information.

File: libs/ui/elements/src/lib/image-overlay-preview/image-overlay-preview.component.html (hunk 2 above)
Comment (cmoinier) [reply libs/ui/elements/src/lib/image-overlay-preview/image-overlay-preview.component.html:2]: Basically, it's like the image is hidden by a mask and there is a "loading animation"... Also, the console shows no error, so I'm a bit at lost on what is wrong here.

But maybe I'm missing something, like a parameter ? Here I have added back the "content-ghost" and its class `ghostClass="h-36 mb-3"`. 

image

**Comment by jahow** [comment]
I amended this PR to address the issues, we changed the position of the md quality indicator like so:
image

**Review by jahow — APPROVED** [review]
